lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -42,11 +42,11 @@ def engine_running(engine, &block)
           end
         end
 
-        def failed(output)
-          spinner.stop("Failed")
-          puts colorize("\nAnalysis failed with the following output:", :red)
+        def errored(output)
+          spinner.stop("Errored")
+          puts colorize("\nAnalysis errored with the following output:", :red)
           puts output
-          exit 1
+          exit 2
```

Comment:
Oh, got it. You're quite right about `G`. I think `E` timeouts vs errors would still be CLI territory: CLI already knows about timeouts & in the "thin-builder" world I think that might continue to be the case?

Yeah, not relevant for this PR anyway: just starting with `170` here is plenty.